openwisp_monitoring/check/tests/test_models.py

```diff
@@ -274,3 +283,28 @@ def test_device_unknown_no_config_check(self):
         self.assertEqual(Metric.objects.count(), 0)
         self.assertEqual(Notification.objects.count(), 0)
         self.assertIsNone(c2.perform_check())
+
+    def test_config_check_problem_with_interval(self):
+        self._create_admin()
+        d = self._create_device(organization=self._create_org())
+        self._create_config(device=d)
+        check = Check.objects.filter(check_type=self._CONFIG_APPLIED).first()
+        self.assertEqual(Check.objects.count(), 3)
+        self.assertEqual(Metric.objects.count(), 0)
```

Comment:
I have tried to run this test without the changes above and it fails here, which doesn't make sense, please make sure the test fails at the right place without the fix.

Reply:
I find out that when we run specific tests, the general metrics don't exist, so it fails at line `293`; when we run all class tests together, it fails at the right place. This behaviour can be seen all over `test_models.py`, where we assert `Metric.objects.count()`. For now, I have changed the assert condition for this test case so that it fails at the right place.